homeassistant/components/alarm_control_panel/device_trigger.py

```diff
@@ -129,14 +138,17 @@ async def async_attach_trigger(
     elif config[CONF_TYPE] == "disarmed":
```

Comment:
@balloob for the "triggered":
Schould I not just remove the `from_state = STATE_ALARM_PENDING`?
Some platforms like the Manual Alarm Panel can be configured to jump imediatly from an armed state to the `triggered` state, withouth the STATE_ALARM_PENDING. (if you do not want a delay between a detection and the alarm going off).